mesh_generation/generate_mesh.py

```diff
@@ -502,7 +502,7 @@ def main():
     )
     parser.add_argument(
         "--wrap-lons",
-        default=False,
+        default=True,
         action="store_true",
```

Comment:
I don't think this is correct.

`action="store_true"` means that
- If the user does not provide `--wrap-lons`, the value is False.
- If the user does provide `--wrap-lons`, the value becomes True.

I think you'll actually need to change the flag, e.g. `--dont-wrap-lons`